src/table/utils/selections-utils.ts

```diff
@@ -129,7 +100,7 @@ export const getSelectedRows = (
   if (evt.shiftKey && key.includes('Arrow')) {
```

Comment:
Will most likely keep the Arrow string here. Creating a sperate array to test this in one place seems overkill to me